### PR TITLE
Proteus: Thread pool improvements.

### PR DESCRIPTION
**Pull request by noboomu**

### CODE REVIEW

File: frameworks/Java/proteus/pom.xml [hunk 1]
```diff
@@ -214,54 +200,51 @@
...
 			<artifactId>mysql-connector-java</artifactId>
 			<version>5.1.47</version>
 		</dependency>
-	  	<dependency>
+		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<version>42.2.5</version>
-		</dependency>  
+			<version>9.4.1212</version>
```
Comment (zloster) [inline frameworks/Java/proteus/pom.xml]: @noboomu Hi, is there a particular reason for downgrading the PostgreSQL JDBC driver?
Some time ago I've made a PR to unify the DB driver versions.

If you have found performance regressions you could try to report them. Some time ago the maintainers implemented basic test suite for this. The project lives here: https://github.com/pgjdbc/benchmarks

Comment (noboomu) [reply frameworks/Java/proteus/pom.xml]: @zloster I am seeing a 25% performance improvement, but I will investigate with the benchmark you linked. Thanks!

Comment (noboomu) [reply frameworks/Java/proteus/pom.xml]: BTW I deleted my fork by accident... should I close this PR and open another?

**Comment by nbrady-techempower** [comment]
I’m going to close this for now. Feel free to reopen after further investigation.

File: frameworks/Java/proteus/pom.xml (hunk 1 above)
Comment (zloster) [reply frameworks/Java/proteus/pom.xml]: Thanks for the response. A while ago I've played with the 9.4.1212 vs 42.0.0. My findings back then are here: https://www.postgresql.org/message-id/d5d39fffac03727934f262c793899321@edno.moe

**Pull request closed without merging**